tools/depends/configure.ac

```diff
@@ -346,19 +361,22 @@ case $host in
           use_sdk="${use_sdk:-$found_sdk_version}"
           sdk_name=$target_platform$use_sdk
           platform_min_version="$target_platform-version-min=9.0"
-        fi
-        case $use_sdk in
-          9.*);;
-          10.*);;
-          11.*);;
-          12.*);;
+
+          case $use_sdk in
+            9.*);;
+            10.*);;
+            11.*);;
+            12.*);;
             *)
               AC_MSG_ERROR(error in configure of --with-sdk=$use_sdk)
-            ;;
-        esac
+              ;;
+          esac
+
+        fi
```

Comment:
I would avoid duplicating the code which is valid for iOS and tvOS, see https://github.com/xbmc/xbmc/compare/master...Rechi:depends/appletvos.

Reply:
I copy paste your changes, thank you.